torchtext/__init__.py

```diff
@@ -1,21 +1,15 @@
 import os
```

Comment:
off-topic, but it looks strange that the top-level module initialization script, which should be only doing initializations require os and do some stuff. These should be done in a dedicated submodule and only imported here.

Reply:
Let me do this in a followup PR. Maybe we can define all global constants in a `_constants.py` file. Wdyt @parmeet?